packages/embed/package.json

```diff
@@ -14,7 +14,7 @@
     "lint:fix": "eslint --cache --fix --ext=js,jsx,ts,tsx src",
     "lint": "eslint --cache --ext=js,jsx,ts,tsx src",
     "lint:env": "dotenv-linter",
-    "verify": "concurrently \"npm:lint:fix\" \"npm:lint:env\""
+    "verify": "concurrently \"npm:lint\" \"npm:lint:env\""
```

Comment:
Great change 👍

Reply:
Will this stop CI from fixing lint errors and returning green anyway?

Reply:
Yes exactly!